content/articles/oop-in-java-abstraction-and-encapsulation/index.md

```diff
@@ -380,23 +348,25 @@ public class Main{
  
  public static void main(){
  
-        Car car = new Benz();
-    
-        car.start();
-    
-        car.stop();
-    }
+     Car car = new Benz();
+ 
+     car.start();
+ 
+     car.stop();
+ 
+ }
+ 
 }
  
 ```
  
 In the main method, `Car` represents the interface class, and `Benz` represents the concrete class or child class. The child class gives its memory to the `Car` object to access its implemented methods. The `Benz` class implemented the start and stop behaviors of the `Car` class in the manner it wanted. This makes our application robust. We can change or add requirements to our application without breaking down the application.
  
-For instance, we want to add another car that stops or starts differently. We can create a car e.g `Toyota` that implements class `Car`, overrides its `stop()` and `start()` methods, and implements it the way we want our `Toyota` to start and stop. There is no need to go into the `Car` class to change our code whenever there are new requirements.
+For instance, we want to add another car that stops or starts differently. We can create a car e.g `Toyota` that implements class `Car`, overrides its `stop()` and `start()` methods, implements them the way we want our `Toyota` to start and stop. There is no need to go into the `Car` class to change our code whenever there are new requirements.
```

Comment:
> We can create a car e.g `Toyota` that implements class `Car`...

Format this like so:

> We can create a car, e.g. `Toyota`, that implements class `Car`, ...

Reply:
@john-amiscaray I am done with the corrections.

Reply:
I will have a look at it very soon.